src/workloads/execution/SecondaryReadsGenny.yml

```diff
@@ -0,0 +1,231 @@
+SchemaVersion: 2018-07-01
+Owner: "@TODO"  # what should go here? us or the owners of secondary_reads?
```

Comment:
Seems like the owners of the workload, so feel free to add Repl if we will be the future owners!

Reply:
Added repl as owners, but feel free to reach out if there are ever any issues questions around the genny stuff :)